gh_actions_exporter/metrics.py

```diff
@@ -187,6 +187,6 @@ def handle_job_duration(self, webhook: WebHook):
                         - webhook.workflow_job.started_at.timestamp())
             self.job_duration.labels(**labels).observe(duration)
         elif webhook.workflow_job.status == "in_progress":
-            duration = (webhook.workflow_job.steps[0].completed_at.timestamp()
+            duration = (webhook.workflow_job.steps[0].started_at.timestamp()
```

Comment:
Ah great catch! perfect.

I do have a strange deja vue feeling that I did this before and was always getting a result close to 0. And it was not representing the actual wait time. But it's been a while and I could have made another mistake so I'll take this.

In case we need to switch things around we can gather this metric only once the job is done, it sucks as we won't get the result in real time, but at least we'll keep a trace of it:
something like:
```python
         elif webhook.workflow_job.conclusion is not None:
             duration = (webhook.workflow_job.steps[0].completed_at.timestamp()
                      - webhook.workflow_job.started_at.timestamp())             
```